client/src/pages/LeftSideBar.js

```diff
@@ -49,11 +50,14 @@ const LeftSideBar = () => {
   const classes = useStyles();
   const history = useHistory();
 
-  const onClickHandler = event => {
-    event.preventDefault();
-    localStorage.clear();
-    history.push("/login");
+  const handleLogout = () => {
+    let response = axios.post("http://localhost:4000/logout");
+    if (response.data.success) {
```

Comment:
This code inside of the if statement should be inside of a `.then` from your above axios call, otherwise every time you hit logout, the `response.data.success` will be undefined since the axios promise call has not resolved yet 

It could be changed to something like this:

```
const handleLogout = () => {
    axios.post("http://localhost:4000/logout").then(() => {
      localStorage.clear();
      history.push("/login");
    });
  };
```

Reply:
Meant to actually use async await! I'll fix that